packages/react/src/components/table/table.tsx

```diff
@@ -201,6 +230,7 @@ export interface TableProps<T extends object> {
     className?: string;
     /** Array of Objects that defines your table columns.
      * See stories code or refer to react-table docs for more information */
+    colSpan?: string;
```

Comment:
You should move the comment because it's now describing the colSpan prop

Reply:
done